### ISSUE TITLE
Title shown in the header

### ISSUE THREAD

**Issue by barotyson** [dediagency/sylius-seo-plugin#20]
After override the layout.html.twig file, i have this bug: 
![immagine](https://user-images.githubusercontent.com/24854557/133466342-9ca74799-edc2-48e4-903a-7281ed9cde11.png)

the products name (Frizione Energizzante Scalpo) is shown in the header bar (even in the title)
 and the title in the header is inserted before the event.

any suggestion?
thanks

**Comment by FlorentinGarnier** [comment]
Cause of `_title.html.twig` of plugin and displaying of `<title>`. You can override this file in your template and correct the behavior.

**Comment by Justin-Jordan** [comment]
hello, I have the same issue and, because I am new to sylius I don't understand your explanation. Can you explain it for a newbie please

**Comment by Justin-Jordan** [comment]
I found the solution. You just have to overide `SyliusShopBundle/Product/show.html.twig` and remove the `block title` on it.

**Comment by JulienLoison** [comment]
Hello

There's not an easy fix for that as we have to declare the `<title>` tag in the event to avoid the debug comments to be rendered in the title. This conflicts with Sylius's usage of calling `parent()` in every block titles. 

https://github.com/dediagency/sylius-seo-plugin/pull/56 does not solve the problem but explains it and documents some solutions to fix the problem after installing the plugin.

**Comment by Healkiss** [comment]
> A simple solution is to replace the title block in twig layout with something like this :
> 
> ```
> {% set app_title_block_output %}
>     {% block title %}
>         {{ sylius_template_event('dedi_sylius_seo_plugin.title', { resource: taxon ?? product ?? page ?? sylius.channel, page: page ?? null }) }}
>     {% endblock %}
> {% endset %}
> <title>{{ app_title_block_output|spaceless|striptags|raw }}</title>
> ```

Imo It should be in the documentation, it avoid to rewrite all the layouts.
Thanks a lot for the problem resolution and the plugin !
